docs/developer-guide/working-with-rules.md

```diff
@@ -396,7 +399,8 @@ module.exports = {
             unnecessaryEscape: "Unnecessary escape character: \\{{character}}.",
             removeEscape: "Remove the `\\`. This maintains the current functionality.",
             escapeBackslash: "Replace the `\\` with `\\\\` to include the actual backslash character."
-        }
+        },
+        hasSuggestions: true
```

Comment:
We include a note about the `fixable` property under the "Applying Fixes" headline on line 307. Can you add a similar `hasSuggestions` note under the "Providing Suggestions" header? Ideally someone can get everything they need to know from that section without having to read through to the examples in this sub-section.

Reply:
Great idea, added.